src/widget/wtracktableview.h

```diff
@@ -35,7 +34,7 @@ class WTrackTableView : public WLibraryTableView {
     bool hasFocus() const override;
     void setFocus() override;
     void keyPressEvent(QKeyEvent* event) override;
-    void loadSelectedTrack() override;
+    void doubleClickSelectedTrack() override;
```

Comment:
We shouldn't name signals like that. Double clicking only works computers with a mouse. It does not work on touchscreens (double tapping), it does not work on controllers (long press?).

How about something like `triggerSelectedTrackSecondaryAction()` or something like that?

Reply:
Or is this a non-issue because this signal will never be emitted by using a touchscreen or a controller?

Reply:
I simply adjusted to the wording in Preferences > Library > **Track Double-Click Action** so that the connection is clear for developers.

* Controller mappings would use [`[Library], GoToItem`](https://manual.mixxx.org/2.3/en/chapters/appendix/mixxx_controls.html#control-[Library]-GoToItem)
* Keyboards: #4359
* I don't remember testing with a touch screen tbh, no idea how one can interact with the library. Depends on the OS and desktop environment IIRC.

Reply:
suggestion:  activateSelectedTrack

Reply:
yup, perfect!